scout/server/blueprints/variants/views.py

```diff
@@ -255,8 +255,27 @@ def sv_variants(institute_id, case_name):
 
     variants_query = store.variants(case_obj['_id'], category='sv',
                                     query=form.data)
-    data = controllers.sv_variants(store, institute_obj, case_obj,
-                                   variants_query, page)
+
+    data = {}
+    # if variants should be exported
+    if request.form.get('export'):
+        document_header = controllers.variants_export_header(case_obj)
+        export_lines = []
+        # Return max 500 variants
+        export_lines = controllers.variant_export_lines(store, case_obj, variants_query.limit(500))
+
+        def generate(header, lines):
+            yield header + '\n'
+            for line in lines:
+                yield line + '\n'
+
+        headers = Headers()
+        headers.add('Content-Disposition','attachment', filename=str(case_obj['_id'])+'-filtered_sv-variants.csv')
```

Comment:
Shouldn't it be `case_obj['display_name']` here? The customers have no idea what the internal case ids mean..

Reply:
You're right. I'll check and modify also the name of the downloaded file for SNVs if the code is like this one!